docs/docs/welcome.md

```diff
@@ -4,35 +4,159 @@ title: Welcome to Botonic Documentation
+### Omnichannel React Components
+
+Botonic adds an abstraction layer on top of messaging platforms so you don't have to worry about different API specifications. Whenever is possible we unify similar concepts under a simple component that works across different channels. For example, [Messenger Quick Replies](https://developers.facebook.com/docs/messenger-platform/send-messages/quick-replies/) and [Telegram Custom Keyboards](https://irazasyed.github.io/telegram-bot-sdk/usage/keyboards/) work in a very similar way, in Botonic you would just use our [Reply](/docs/components/replies) component like this:
+
+```html
+<Text>
+  What's you favourite color?
+  <Reply payload="purple">Purple</Reply>
+  <Reply payload="orange">Orange</Reply>
+  <Reply payload="other">Other</Reply>
+</Text>
+```
+
+Sometimes messaging apps support elements that don't exist in other platforms so we can't make this straightforward standarization. For instance [Messenger's Carousel](https://developers.facebook.com/docs/messenger-platform/send-messages/template/generic/#carousel) has no equivalent on Whatsapp. In these case you can use our [Multichannel](/docs/components/multichannel) component which offers a sane transformation using available elements in the destination platform, or you can manage it yourself.
+
+In this sense, **Botonic is to messaging platforms what React Native is to mobile platforms**.
+
+Checkout the supoprted [components](/docs/components/components).
+
+### Fully Customisable Webchat SDK
+
+Messaging apps offer a limited set of interactive elements, so there's only so much you can do in terms of UI as you're not in full control. However, if you're building for the web you can use the full power of React. Our [Webchat SDK](/docs/concepts/webchat) allows you to easily customize the behaviour and styles using properties like `enableEmojiPicker: true` or `brandColor: 'blue'`, but also allows for more advanced adaptations with **custom components** and [custom message types](/docs/concepts/webchat#custom-message-styles). With the latter, you can **embed complex UI elements** right in the middle of the conversation, like forms or calendars.
+
+### Webviews
+
+[Webviews](/docs/concepts/webviews) allow you to work around the fact that you can't control messaging apps UI, so you can't embed your custom components in, say, a Whatsapp conversation. However you can open a **browser window as an extension of the current conversation** (aware of the context), let the user perform some complex task there, and return to the messaging UI to continue the conversation. Some platforms like [Facebook Messenger support webviews](https://developers.facebook.com/docs/messenger-platform/webview/) in a native way, but for the rest, Botonic manages it with deep links.
+
+Webviews are great for use cases where you need complex components or when data privacy/security is important, for instance:
```

Comment:
Webviews are great when you need complex components or when data privacy/security is important, for instance: